_data/155.yml

```diff
@@ -0,0 +1,241 @@
+name: Leap
+version: 15.5
+bg-color: leap
+fg-color: black
+logo: logos/leap.svg
+choosing-media: true
+leap-switch: true
+downloads:
+- name: installation_images
+  display: all
+  arches:
+  - name: x86_64
+    types:
+    - name: offline_image
+      desc: offline_desc
+      primary_link: /distribution/leap/15.5/iso/openSUSE-Leap-15.5-DVD-x86_64-Media.iso
+      links:
+      - name: metalink
+        url: /distribution/leap/15.5/iso/openSUSE-Leap-15.5-DVD-x86_64-Media.iso.meta4
+      - name: pick_mirror
+        url: /distribution/leap/15.5/iso/openSUSE-Leap-15.5-DVD-x86_64-Media.iso?mirrorlist
+      - name: checksum
+        url: /distribution/leap/15.5/iso/openSUSE-Leap-15.5-DVD-x86_64-Media.iso.sha256
+      - name: torrent_file
```

Comment:
Do we have torrent files available already? They usually aren't a thing prior to release

Reply:
Worth asking the mirrorcache developers, I know we have intentionally omitted torrent files in previous Alphas and Betas because they weren't available